src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -61,7 +61,19 @@ Then, you can see the error in your dashboard:
 [{% asset js-index/error-message.png alt="Error in Unresolved Issues with title This is my fake error message" %}]({% asset js-index/error-message.png @path %})
 
 ## Capturing Errors
-In most situations, you can capture errors automatically with `captureException()`.
+Automatically and manually capture errors, exceptions, and rejections.
+
+### Automatically Capture Errors
+By including and configuring Sentry, the SDK will automatically attach global handlers to capture uncaught exceptions and unhandled rejections.
+
+[{% asset js-index/automatically-capture-errors.png alt="Stack trace of a captured error" %}]({% asset js-index/automatically-capture-errors.png @path %})
+
+Keep in mind; browsers are taking some security measures when serving script files from different origins. For errors to always make their way to Sentry, make sure that you configure CORS headers and add appropriate script attributes. 
```

Comment:
Agree with Ben that this reads oddly. Suggest "Browsers take security measure when serving script files from different original. To ensure errors are routed to Sentry, configure the CORS headers and add script attributes."

We should also add the content from the blog post, then remove this link since the blog is pretty dated. We can manage that change in a separate PR, if that's better.